### PR TITLE
Delete by query in bulk loader

### PR DESCRIPTION
**Pull request by ghost**
PR #91 needs to be merged before this one.

### CODE REVIEW

File: lib/open-search-client.js [hunk 1]
```diff
@@ -195,6 +195,9 @@ const createTermsIndex = (prefixUrl, index) => createIndex(prefixUrl, index, {
       source_terminology: {
         type: 'keyword',
       },
+      graph_uri: {
```
Comment (paulpilone) [inline lib/open-search-client.js]: Could we camelCase this? I'd like to fix `source_terminology` too if you want to do that in this PR.

File: neptune-bulk-loader/task.ts [hunk 2]
```diff
@@ -73,6 +73,12 @@ export const neptuneBulkLoader = async (): Promise<MainResult> => {
 
   await createTermsIndex(esUrl, termsIndex);
 
+  await osClient.deleteByQuery(esUrl, termsIndex, {
+    match: {
+      graph_uri: metadata.namedGraphUri,
```
Comment (paulpilone) [inline neptune-bulk-loader/task.ts]: If you change this ES field name you'll need to update it here too.

File: lib/open-search-client.js (hunk 1 above)
Comment (ghost) [reply lib/open-search-client.js]: Fixed in 84b7185

File: neptune-bulk-loader/task.ts (hunk 2 above)
Comment (ghost) [reply neptune-bulk-loader/task.ts]: Fixed in 84b7185

**Review by paulpilone — APPROVED** [review]

**Pull request merged**